Mathlib/Topology/Homeomorph.lean

```diff
@@ -867,3 +867,109 @@ def homeoOfEquivCompactToT2 [CompactSpace X] [T2Space Y] {f : X ≃ Y} (hf : Con
 #align continuous.homeo_of_equiv_compact_to_t2 Continuous.homeoOfEquivCompactToT2
 
 end Continuous
+
+variable [TopologicalSpace X] [TopologicalSpace Y] [TopologicalSpace Z]
+  {W : Type*} [TopologicalSpace W]
+
+/-- Predicate saying that `f` is a homeomorphism. -/
+def IsHomeomorph (f : X → Y) := Continuous f ∧ IsOpenMap f ∧ Function.Bijective f
```

Comment:
It's usually nicer to create a custom structure when combining > 2 predicates
```suggestion
structure IsHomeomorph (f : X → Y) : Prop where
  continuous : Continuous f
  isOpenMap : IsOpenMap f
  bijective : Bijective f
```
(also please `open Function`)